tests/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 project(GSLTests CXX)
+
 include(FindPkgConfig)
+include(ExternalProject)
+
+find_package(Git REQUIRED QUIET)
```

Comment:
Please do not require Gti for build tests. It is still possible to get GTest through APT or other package manager even if Git is not installed.

Reply:
The reason I added the requires was because I thought it was required. In a lot of CMake code this is just left out, hence I thought it was just left out here as well.

Also in my CMake workflow I tend to just download things :)

Feel free to make the pull request.

But add a comment explaining why the requires isn't necessary in the new PR. Commenting is very important in CMake.